api-docs/cloud-load-balancers-v1/api-reference/ciphers.rst

```diff
@@ -0,0 +1,71 @@
+.. _ciphers:
+
+==============
+Ciphers 
+==============
+
+.. contents::
+   :local:
+   :depth: 1
+
+Ciphers are algorithms for performing encryption and decryption. They are used
+to help provide secure communications over computer networks. Load balancers
+that make use of the SSL Termination feature are configured to use only
+certain ciphers based on the assigned cipher profile.
+
+Ciphers profiles are a named set of cipher suites to be used by a load balancer.
+The cipher profile can be assigned via the ``cipherProfile`` field
+when creating or updating :ref:`SSL Termination <put-update-ssl-termination-configuration>`
+configuration for the load balancer.
+
+By default, load balancers are assigned the ``default`` cipher profile which is
+managed by Rackspace and updated from time to time to disable ciphers that have
+become insecure. For this reason, use of the ``default`` cipher profile is
+recommended.
+
+.. warning::
+   Only the ``default`` profile is updated to remove insecure ciphers.
+   If you have a load balancer with a cipher profile assigned that uses insecure
+   or weak ciphers, we highly recommended that you assign an updated cipher
+   profile or re-assign the ``default`` profile.
```

Comment:
Question on this warning block.  If the `default` profile is updated over time to remove insecure ciphers, why does the table list `default` as having the 🔥 `3DES` 🔥 cipher in it while `CLBCipherPolicy2017-08` does not?